database/models.py

```diff
@@ -1,17 +1,26 @@
 import uuid
 
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import Enum, func
+from sqlalchemy import Enum, func, String, Column
+from sqlalchemy.orm import DeclarativeBase
 
-db = SQLAlchemy()
 
-
-class Base(db.Model):
+class Base(DeclarativeBase):
     __abstract__ = True  # Indicates that this class should not be created as a table
-    id = db.Column(db.String(36), primary_key=True, default=lambda: str(uuid.uuid4()))
+    id = Column(String(36), primary_key=True, default=lambda: str(uuid.uuid4()))
+
+
+db = SQLAlchemy(model_class=Base)
 
 
-class Organization(Base):
+class Error(db.Model):
```

Comment:
we should include `harvest_job_id` as common to both errors as we will need to correlate record level errors to the job unless i'm missing something.

Reply:
[resolved with](https://github.com/GSA/datagov-harvesting-logic/blob/8baa21616de9693b43ff78e40f8e05d0cfeba56e/database/interface.py#L214)